docs/user-guides/Delegating-CSPR-Stake.md

```diff
@@ -0,0 +1,56 @@
+# Delegating CSPR Stake
+
+## What You'll Need
+
+A compatible wallet with a CSPR balance: either [CasperLabs's Signer](https://chrome.google.com/webstore/detail/casperlabs-signer/djhndpllfiibmcdbnmaaahkhchcoijce) or [Ledger](https://www.ledger.com/). More wallets are planned for the future.
+
+## The Basics
+
+### Connect a Wallet
+
+You can access the wallet Sign In screen by clicking `Sign In` from the top navigation menu. Then choose your Sign In option, either Signer or Ledger:
+
+#### ...Using Signer
+
+1. From the Sign In screen click `Sign In` under the Casper Signer option
+2. The Signer app window will open automatically. From the Signer window, when prompted to connect Signer to site, click `Connect` . Then click `Connect` again to approve the connection.
+3. Select an account to connect to.
+
+#### ...Using Ledger
```

Comment:
No Ledger currently. Do we want to comment all this out so it's available for re-inclusion later?

Reply:
:+1: We will have it in the commit and commenting out in markdown seems a bit hack/messy. I think we can just remove it for now.